### ISSUE TITLE
Jetpack Compose support

### ISSUE THREAD

**Issue by dannyvalentesonos** [rive-app/rive-android#272]
Are there any plans to support Jetpack Compose? Is there any way to do this already today? I've looked, but can't find an example.

**Comment by hiteshchopra11** [comment]
Bump

**Comment by carloshs94** [comment]
There's already an issue with this topic #259

**Comment by delrioe** [comment]
Hey guys, I took the time to find a way to have Rive composables in my project and I made a post in Stack Overflow with the code.
That is the best solution I could come up with.

[Rive + Jetpack Compose](https://stackoverflow.com/questions/77668942/rive-jetpack-compose-android)

Hope it helps.

**Comment by slai47** [comment]
@delrioe thank you for the thread. Looks like it works quiet well but at the same time, the Rive team should really get a full support Compose view in so we don't have to do this. Even if its a wrapper of the XML on their end just so they can update it later.

**Comment by delrioe** [comment]
@slai47  I totally agree with you on that. Hopefully their Android team can grow and offer more support on this area. 
Meanwhile, let me know if you run into issues with that implementation. It should be pretty straight forward and it should handle listeners and state changes.

**Comment by ErikUggeldahl** [comment]
@delrioe Thanks for filling in the gaps as you did on StackOverflow.

Compose support is around the corner, and rather than have multiple open issues tracking it, I will close this one as a duplicate. Please follow #259 on GitHub, but there will also be announcements. Not long now.
